lib/ps.js

```diff
@@ -0,0 +1,80 @@
+var exec = require('child_process').exec
+var os = require('os')
+var PLATFORM = os.platform()
+
+/**
+  * Get pid informations through ps command
+  * @param   {Number|Number[]}  pid
+  * @param   {Object}           options
+  * @return  {Function}         callback(err, stat)
+  * on os x skip headers with pcpu=,rss=
+  * on linux it could be --no-header
+  * on solaris 11 can't figure out a way to do this properly so...
+  */
+function ps (pid, options, done) {
+  var advanced = options.advanced
+  if (!Array.isArray(pid)) {
+    pid = parseInt(pid, 10)
+  } else {
+    advanced = true
```

Comment:
Why?
Anyway I believe that we can completely remove this "advanced" option.
If users needs less info they can map the output object.
This just adds complexity to the code

Reply:
yes I dunno why I kept that